### ISSUE TITLE
Node that triggers once chain is finished running

### ISSUE THREAD

**Issue by radry** [chaiNNer-org/chaiNNer#2476]
Not sure how to call it but it would be nice to have a node that triggers and calls an extern script or program when the current chain is finished running. Additionally it would be good if this node has a "string input" so we can pass data as arguments to the external script or program.

Example use Cases for external scripts/programs that could get triggered:  

delete the original file or directory after upscaling is complete

or 

move output files from temporary directory to local or remote storage

or

send notification of completion
